.github/scripts/otp-compliance.es

```diff
@@ -551,6 +540,22 @@ fix_beam_licenses(LicensesAndCopyrights,
                               %% follows from otp/lib/stdlib/uc_spec/README-UPDATE.txt
                               files_have_no_license(SPDX#{~"licenseConcluded" := ~"Unicode-3.0 AND Apache-2.0"});
 
+                          #{~"fileName" := ~"erts/emulator/zstd/COPYING"} when
```

Comment:
Why does this need to be here and in `.ort.yml`? Shouldn't `.ort.yml` be enough?

Reply:
Right, normally yes. however, our clients would like to have a SPDX comment that mentions that this file specifically is a license file. The `.ort.yml` cannot encode this, and we must address it in this compliance script